compiler/core/js_name_of_module_id.ml

```diff
@@ -31,7 +31,7 @@ let string_of_module_id_in_browser (x : Lam_module_ident.t) =
   match x.kind with
   | External {name} -> name
   | Runtime | Ml -> 
-    "./stdlib/" ^  Ext_string.uncapitalize_ascii x.id.name ^ ".js"
+    "./stdlib/" ^ x.id.name ^ ".js"
```

Comment:
are you sure this can go away for both Ml and Runtime?

Reply:
I think this is only used in the playground so it should be safe. It was invalid before.